.vaunt/config.yaml

```diff
@@ -0,0 +1,185 @@
+          name: docs_dash_pr_merged
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-26 20:00:00
+                        & labels in {'2024-tidb-docs-dash','contribution'}"
+                    value: 10
+    - action:
+          type: point
+          name: docs_dash_issue_contribution
+          triggers:
+              - trigger:
+                    actor: author
+                    action: issue_comment
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & labels in {'2024-tidb-docs-dash'}
+                        & emojis in ['THUMBS_UP']
+                        & reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']"
+                    value: 2
+    - action:
+          type: point
+          name: issue_contribution
+          triggers:
+              - trigger:
+                    actor: author
+                    action: issue_comment
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & emojis in ['THUMBS_UP']
+                        & reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']
+                        & labels in !['2024-tidb-docs-dash']"
+                    value: 1
+    - action:
+          type: point
+          name: pr_submitted
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & labels in !['2024-tidb-docs-dash']
+                        & labels in ['contribution']"
+                    value: 2
+    - action:
+          type: point
+          name: pr_merged
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-26 20:00:00
+                        & labels in !['2024-tidb-docs-dash']
+                        & labels in ['contribution']"
+                    value: 5
+    - action:
+          type: point
+          name: issue_created
+          triggers:
+              - trigger:
+                    actor: author
+                    action: issue
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
```

Comment:
@elewis787 I think we also need to add 
`& emojis in ['THUMBS_UP']
& reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']` 
so that we can validate that these are valid contributions?

Reply:
I'll update - we will need to add a issue_comment action through a & operator for that - as emojis are only available on the issue comments.

Reply:
updated!